src/LiveComponent/doc/index.rst

```diff
@@ -2248,6 +2248,71 @@ Or, in PHP::
 
     $this->emitSelf('productAdded');
 
+Dispatching Browser/JavaScript Events
+-------------------------------------
+
+Sometimes you may want to dispatch a JavaScript event from your component. You
+could use this to signal, for example, that a modal should close::
+
+    use Symfony\UX\LiveComponent\ComponentToolsTrait;
+    // ...
+
+    class MyComponent
+    {
+        use ComponentToolsTrait;
+
+        #[LiveAction]
+        public function saveProduct()
+        {
+            // ...
+
+            $this->dispatchBrowserEvent('modal:close');
+        }
+    }
+
+This will dispatch a ``modal:close`` event on the top-level element of
+your component. It's often handy to listen to this event in a custom
+Stimulus controller - like this for Bootstrap's modal:
+
+.. code-block:: javascript
+
+    // assets/controllers/bootstrap-modal-controller.js
+    import { Controller } from '@hotwired/stimulus';
+    import { Modal } from 'bootstrap';
+
+    export default class extends Controller {
+        modal = null;
+
+        initialize() {
+            this.modal = Modal.getOrCreateInstance(this.element);
+            window.addEventListener('modal:close', () => this.modal.hide());
+        }
+    }
+
+Just make sure this controller is attached to the modal element:
+
+.. code-block:: html+twig
+
+    <div class="modal fade" {{ stimulus_controller('bootstrap-modal') }}">
```

Comment:
Redundant quote at the end?

Reply:
Absolutely - thanks for mentioning that.